scripts/pioarduino_env.py

```
@@ -0,0 +1,47 @@
+import os
+
+Import("env")  # noqa: F821 — provided by PlatformIO
+
+env.Append(CXXFLAGS=["-Wno-deprecated-enum-enum-conversion"])
```

Ruff (0.15.10)

[error] Undefined name `env`

(F821)